andela/travel_tool_back
# #160597069 Add guest room and bed during request creation

#### What does this PR do?

This PR ensure that user enters available room and bed space during new travel request creation.

#### Description of Task to be completed?

As a user, I should be able to add the room and bed space during my new travel request. This will ensure that if I will not get any room available, they will look for alternatives or postpone trip duration. 

#### How should this be manually tested?
- git clone https://github.com/andela/travel_tool_back.git
- git checkout ft-check-available-room-bed-160597069
- yarn install
- make rollback && make migrate
- make seed
- Open postman and provide your token in the authorization header. Enter url POST  for making a request: http://localhost:5000/api/v1/requests/ and make a request using [this JSON Data](https://gist.github.com/ezrqnkemboi/ac49853543e794de425e480fcc15b492)

#### Any background context you want to provide?
Ensure that before creating a request, you should create a guest house using [this PR](https://github.com/andela/travel_tool_back/pull/65)
Another point to note is that, the location in the seeded data is Lagos,Nigeria. What comes from frontend is Lagos, Nigeria or Nairobi, Kenya for the case of Nairobi. 
So, My issue is that if this work is to be demoed with the seeded data, then , the raw for guest house location need to be Lagos, Nigeria or Nairobi, Kenya, so that it will bed will not be available. 

#### What are the relevant pivotal tracker stories?

[#160597069](https://www.pivotaltracker.com/story/show/160597069)

#### Screenshots (if appropriate)

<img width="1153" alt="screen shot 2018-10-18 at 20 33 05" src="https://user-images.githubusercontent.com/32410518/47172775-0c5d1180-d315-11e8-929a-8a73a603171f.png">



#### Questions && Concern:
If, seeded data is used to make a request, then the place for location of guest house need to change to Lagos, Nigeria or Nairobi, Kenya. I have handled the spacing when making a query from the frontend.
